jsonpickle/unpickler.py

```diff
@@ -19,7 +19,7 @@ def decode(
     context=None,
     keys=False,
     reset=True,
-    safe=False,
+    safe=True,
```

Comment:
I have a gut feeling that this is going to come back to bite us in that loadmodule isn't compatible with some use case, but I can't find evidence for this feeling so YOLO I guess.

Reply:
Yeah, this is probably the part that is worth a major version bump just for extra safety.

The question I faced here was, "should jsonpickle be safe by default, or opt-in to safety?". The correct answer should really be "safe by default," but it meant changing semantics. It's hard to know if anyone was relying on it so the changelog will need to be updated to call this out more.

Unfortunately we won't find out if this affects anyone until we put it out there and wait 3-5 years for all the slow upgraders to come out of the woodwork. As long as we have an easy way (`safe=False`) for users to opt-in to the compatible behavior then we should end up in a better place as they'll be consciously choosing to use the unsafe mode.

Reply:
I think you're right, jsonpickle should absolutely be safe by default. I do think that before this is merged, the CHANGES.rst file should be updated to specify that `safe=False` is an alternative if the user so chooses. I'll go ahead and update that now, and you can feel free to edit it and then merge it if you want.